docker/build_dagmc.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+source ${docker_env}
+
+if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+  build_mw_reg_tests=ON
+else
+  build_mw_reg_tests=OFF
+fi
+
+rm -rf ${dagmc_build_dir}/bld ${dagmc_install_dir}
+mkdir -p ${dagmc_build_dir}/bld
+cd ${dagmc_build_dir}
+#git clone https://github.com/svalinn/DAGMC -b develop
```

Comment:
Recommend leaving a clarifying comment for commented out code, both here and on line 32.  Ideally the comment should elaborate on:

1. Reasons for commenting out code
2. Under what conditions it can be re-enabled or removed.